addon/components/tinymce-ember.js

```diff
@@ -126,7 +126,7 @@ export default Component.extend({
         this.bindEditorCustomEvents(Editor);
       }
 
-      if (this.content !== this.currentContent) {
+      if (this.content && this.content !== this.currentContent) {
```

Comment:
I don't agree with this condition.
If `this.content === ''` you're not able to reset the editor content.

Reply:
```suggestion
      if (this.content ?? this.content !== this.currentContent) {
```
Pour checker les valeurs nullish et pas falsey ?